### PR TITLE
Aria2 crash fix

### LINKED ISSUE #272: If aria2c can not be started, print the command line which has failed
Following #268, it looks like this is pretty difficult to understand what is going on. If the aria2c downloader can not be setup, which basically means that the aria2c binary has not been launched successfully, the full command should be printed somewhere (on the terminal for example) so we get a chance to understand why it failed.

### PR DESCRIPTION
**Pull request by jetownfeve21**
display aria2 launch cmd that doesn't crash

The program crashes because it tries to use mp_downloader->getAria2LaunchCmd() but the mp_downloader is null if the aria2 launch command fail.
Now the program displays an error message to warn users that all download fct will not working.

This PR https://github.com/kiwix/kiwix-lib/pull/306 is needed in order to display the aria2 launch cmd.

fix #272

### CODE REVIEW

File: src/kiwixapp.cpp [hunk 1]
```diff
@@ -11,31 +11,25 @@
...
+      mp_downloader(nullptr),
       mp_server(new kiwix::KiwixServe(
         appendToDirectory(m_libraryDirectory.toStdString(),"library.xml"),
         m_settingsManager.getKiwixServerPort()))
 {
+    try {
+        mp_downloader = new kiwix::Downloader();
+    } catch (exception& e) {
+        QMessageBox::critical(nullptr, tr("Cannot create downloader"),
+        tr("Impossible to launch downloader, Kiwix-desktop will start but all download functions will not working !<br><br>") + e.what());
```
Comment (mgautierfr) [inline src/kiwixapp.cpp]: `tr` will try to load the localized text. So we should install the translators before.

File: src/kiwixapp.cpp [hunk 2]
```diff
@@ -115,6 +105,9 @@ KiwixApp::~KiwixApp()
         mp_downloader->close();
         delete mp_downloader;
     }
+    if (mp_manager) {
+        delete mp_manager;
+    }
```
Comment (mgautierfr) [inline src/kiwixapp.cpp]: mp_manager should always exists. No need to test if mp_manager is not null

File: src/kiwixapp.cpp (hunk 1 above)
Comment (jetownfeve21) [reply src/kiwixapp.cpp]: We will not use the tr() fct to translate, I want to remove all the "hard coded" text in another PR to fix https://github.com/kiwix/kiwix-desktop/issues/299 and use a translate file instead to work with translateWiki. This solution should also make it possible to better manage translations in HTML/JS files such as the contentmanager page.

Comment (mgautierfr) [reply src/kiwixapp.cpp]: Ok, I don't know how translateWiki works. But it will probably use a function to get the text
to display to the user. And this function must be used after the configuration of the
translation system.

And for now, we are using `tr`. We must move this after the installation of the translators
anyway. At least to have the translation working while we are moving to translateWiki.

**Comment by kelson42** [comment]
@jetownfeve21 Could you please finish this PR?

**Comment by mgautierfr** [comment]
We are good !
